src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,6 +1,11 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
 public class CinemaHallRequestDto {
+    @NotNull
+    @Min(value = 0)
     private int capacity;
```

Comment:
```suggestion
    @Min(value = 0)
    private int capacity;
```
int will never be null